.taprc

```diff
@@ -55,6 +55,7 @@ files:
   - ./packages/cactus-test-cmd-api-server/src/test/typescript/integration/remote-plugin-imports.test.ts
   - ./packages/cactus-test-cmd-api-server/src/test/typescript/integration/plugin-import-with-npm-install.test.ts
   - ./packages/cactus-test-cmd-api-server/src/test/typescript/integration/plugin-import-with-npm-install-version-selection.test.ts
+  - ./packages/cactus-plugin-ledger-connector-corda/src/test/typescript/integration/openapi/openapi-validation.test.ts
```

Comment:
@awadhana The PR title says `test: jestify runtime-plugin-import test #1590` but the test here is `openapi-validation.test.ts` is the title incorrect or the diff here has an issue?

Reply:
Changes have been made for this